lib/pensio_api/altapay_version.rb

```diff
@@ -0,0 +1,10 @@
+module PensioAPI
+
+    class AltapayVersion
+
+        def self.getVersion
+        #add here the latest version (typically will be the one that will be released after pushing the latest changes)
+            version = '0.3.4'
```

Comment:
What is the function of this local variable? Assignment here is doing nothing, and the variable falls out of scope immediately. You'd be better just returning the version string, or, as mentioned, make this a constant:

```module PensioAPI

  ALTAPAY_VERSION = '0.3.4'.freeze
  
end```

Reply:
In fact, you don't need this class at all, just use the constant `PensioAPI::VERSION` defined lib/pensio_api/version.rb